src/components/DigitalCredentials/dialogs/ConfirmReplaceCredentialDialog.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <v-dialog
+    v-model="dialog"
+    width="45rem"
+    persistent
+    :attach="attach"
+    content-class="confirm-replace-credential-dialog"
+  >
+    <v-card class="pa-5">
+      <v-card-subtitle
+        class="pt-5"
+        align="center"
+      >
+        <v-icon
+          color="error"
+          large
+        >
+          mdi-information-outline
+        </v-icon>
+        <v-btn
+          small
+          text
+          color="primary"
+          class="close-btn"
+          @click="close()"
+        >
+          <v-icon>mdi-close</v-icon>
+        </v-btn>
+      </v-card-subtitle>
+
+      <v-card-text>
+        <p
+          id="dialog-title"
+          class="warning-title"
+        >
+          Are you sure?
+        </p>
+        <p
+          id="dialog-text"
+          class="warning-text"
+        >
+          This will replace the credential in your BC Wallet. It cannot be reversed. Do you still want to continue?
+        </p>
+      </v-card-text>
+
+      <v-card-actions>
+        <v-row
+          no-gutters
+          justify="center"
+        >
+          <v-btn
+            id="dialog-close-button"
+            class="mr-4 action-btn"
+            color="primary"
+            outlined
+            @click="close()"
+          >
+            <span>Cancel</span>
+          </v-btn>
+          <v-btn
+            id="dialog-proceed-button"
+            class="action-btn"
+            color="primary"
+            @click="proceed()"
+          >
+            Continue with Credential Replacement
+            <v-icon>mdi-chevron-right</v-icon>
+          </v-btn>
+        </v-row>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
+
+@Component({})
+export default class ConfirmReplaceCredentialDialog extends Vue {
+  /** Prop to display the dialog. */
+  @Prop({ default: false }) readonly dialog!: boolean
+
+  /** Prop to provide attachment selector. */
+  @Prop({ default: '' }) readonly attach!: string
+
+  // Pass click events to parent.
+  @Emit() close (): void {
+    return undefined
+  }
+  @Emit() proceed (): void {
+    return undefined
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.close-btn {
+  position: absolute;
+  right: 2rem;
+}
+
+.action-btn {
+  min-height: 44px;
+}
+
+.warning-title {
+  font-size: 1.5rem;
```

Comment:
For fonts, use the CSS variables in theme.scss. Here, it would be `$px-24`.

Reply:
For this and my other comments, if they apply to other files then please fix there also (so I don't have to repeat these comments). Thanks!

Reply:
Updated the theme file with additional sizes

Reply:
Looks good.

As mentioned elsewhere, I'm OK with using these "font" variables for line heights since they're related.